<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newform</class>
 <widget class="QWidget" name="newform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>457</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 245, 245);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;更纱黑体 UI SC&quot;;</string>
            </property>
            <property name="text">
             <string>排序模式</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="colorType">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;更纱黑体 UI SC&quot;;</string>
            </property>
            <item>
             <property name="text">
              <string>亮度</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>权重</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>平均</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;更纱黑体 UI SC&quot;;</string>
            </property>
            <property name="text">
             <string>颜色数量</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="colorNum">
            <property name="minimumSize">
             <size>
              <width>44</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Source Code Pro&quot;;</string>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="startBtn">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;更纱黑体 UI SC&quot;;</string>
            </property>
            <property name="text">
             <string>开始</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="saveBtn">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;更纱黑体 UI SC&quot;;</string>
            </property>
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="editBtn">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;更纱黑体 UI SC&quot;;</string>
            </property>
            <property name="text">
             <string>格式</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="loadBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>19</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;更纱黑体 UI SC&quot;;</string>
            </property>
            <property name="text">
             <string>打开</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGraphicsView" name="imgShow">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>457</width>
        <height>340</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;border:0px</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
